app/controllers/builder_controller.rb

```diff
@@ -71,19 +72,32 @@ def caption_params
   end
 
   def create_charge
+    charge = process_stripe_charge
+    postcard.update_attribute(:stripe_charge_id, charge[:id])
+    flash[:notice] = 'Your photo has been sent!'
+  rescue StandardError
```

Comment:
Stripe provides a few different error classes to help us give more informed error messages, but I didn't want to invest in that yet.